app/main/helpers/validation.py

```diff
@@ -282,6 +282,27 @@ def errors(self):
         return errors_map
 
 
+class DOS5Validator(SharedValidator):
+    """Following an accessibility review, a number of questions and answers were changed for DOS 5"""
+    email_validation_fields = {"contactEmailContractNotice", "contactEmail"}
+
+    number_fields = [("subcontracting30DayPayments", 0, 100)]
+
```

Comment:
Are you planning to add the validation for the subcontracting logic in alphagov/digitalmarketplace-frameworks#633 in a separate PR?